v2/pkg/catalog/loader/remote_loader.go

```diff
@@ -55,7 +56,27 @@ func getRemoteTemplatesAndWorkflows(templateURLs []string, workflowURLs []string
 	return remoteTemplateList, remoteWorkFlowList, err
 }
 
-func getRemoteContent(URL string, w chan<- RemoteContentError, contentType ContentType) {
+func getRemoteContent(URL string, remoteTemplateDomainList []string, w chan<- RemoteContentError, contentType ContentType) {
+	if strings.HasPrefix(URL, "http") && (strings.HasSuffix(URL, ".yaml") || strings.HasSuffix(URL, ".yml")) {
+		parsed, err := url.Parse(URL)
```

Comment:
please state exactly what a variable holds. It's a `parsedURL`, not simply just "parsed"